### ISSUE TITLE
Windows standalone build

### ISSUE THREAD

**Issue by gdurif** [diyabc/diyabcGUI#20]
1. run `build_standalone.R` (based on R package `electricShine`)
2. If 1. not working, try running `build_standalone_windows.R` (based on R package `RInno`)

**Comment by fradav** [comment]
Both scripts fails with : 
```plain
package 'rlang' successfully unpacked and MD5 sums checked
Erreur : (converti depuis l'avis) cannot remove prior installation of package 'rlang'
Exécution arrêtée
```

**Comment by gdurif** [comment]
`build_standalone.R` with `electricShine` worked, c.f. https://github.com/diyabc/diyabcGUI/releases/tag/v1.0

In the future, we might want to switch to `build_standalone_windows.R` (potential cleaner build)

**Comment by fradav** [comment]
Apparently the standalone is missing a few key DLL for the included binaries to work on a "vanilla" windows : 
- libiomp5md.dll (openmp library used by diyabc, distributed with intel mkl installation)
- vcruntime140_1.dll (microsoft c++ redistributable). 
I make two separate issues about this

**Comment by fradav** [comment]
https://github.com/diyabc/abcranger/issues/64
https://github.com/diyabc/diyabc/issues/36

As a side note, we should have a "vanilla" windows VM (as a snapshot for example) to test this.

**Comment by fradav** [comment]
Good news, I can reproduce the problems in a windows vm.

**Comment by fradav** [comment]
It is completely fixed for abcranger latest release.
However, for diyabc exe, the choice of openmp as multithreading backend forces us to deploy the windows binary with two dlls (included in the release page) vcomp140.dll and vcruntime140.dll : there is no workaround, sorry.

**Comment by gdurif** [comment]
**TODO**
- find an alternative to `electricShine` to only wrap R and package dependencies inside the binary and not a pseudo-browser, i.e. the standalone would run `diabcGUI::diyabc()` and open the app in the host web-browser.

**Comment by gdurif** [comment]
Change of plan and new strategy based on DesktopDeployR: see 4178965

TODO:
- fix issue with log files
- fix issue when running multiple instances simultaneously

**Comment by gdurif** [comment]
TODO fixed in current release (4fd8d59).

**Comment by gdurif** [comment]
New standalone release strategy based on DesktopDeployR is now running smoothly.

See `build/README.d` and `build/DesktopDeployR/README.md` for more details and instructions.
